tools-v2/README.md

```diff
@@ -1057,6 +1057,30 @@ Output:
 +-------------------+-------------------+-------------------+----------+
 ```
 
+#### staus snapshot
+
+get the snapshot clone status of curvefs
+
+Usage:
+
+```bash
+curve bs status snapshot
```

Comment:
@Cyber-SiKu we'd better use a more meaningful cmd, which can distinguish volume snapshot progress and snapshot servers' status, and raft snapshot.